lib/internal/parameterPopulation.ts

```diff
@@ -17,7 +17,7 @@ import { Arg } from "./invoker/Payload";
  * @param args string args
  */
 export function populateParameters(instanceToPopulate: any, hm: CommandHandlerMetadata, args: Arg[]) {
-    args.forEach(arg => {
+    (args || []).forEach(arg => {
```

Comment:
Good check, but that means someone explicitly defined that parameter as undefined? I would make this param optional then.

Reply:
We are using the same pattern in that code already. That's why I went with this. Just staying consistent-